emcc.py

```diff
@@ -3295,6 +3299,10 @@ def phase_final_emitting(options, state, target, wasm_target, memfile):
   if options.executable:
     make_js_executable(js_target)
 
+  if options.embind_emit_tsd:
+    out = shared.run_js_tool(js_target, [], stdout=PIPE)
```

Comment:
Its pretty nice how easy this is to run the output of the compiler and read it back in again like this!